website/src/docs/hotchocolate/defining-a-schema/extending-types.md

```diff
@@ -18,176 +16,340 @@ extend type Query {
 }
 ```
 
-> **Note:** Every single code example will be shown in three different approaches, annotation-based (previously known as pure code-first), code-first, and schema-first. However, they will always result in the same outcome on a GraphQL schema perspective and internally in Hot Chocolate. All three approaches have their pros and cons and can be combined when needed with each other. If you would like to learn more about the three approaches in Hot Chocolate, click on [Coding Approaches](/docs/hotchocolate/api-reference/coding-approaches).
+In Hot Chocolate type extensions work a little different. Instead of generating the `extend` syntax in the schema, the original type definition and its type extensions are merged at runtime into a combined type.
+
+Type extensions make most sense, if
+
+- we want to split up types into separate classes.
+
+- we can not modify the original type.
+
+> Note: If we just want to organize the fields of one of our types in different files, we can use [partial classes](https://docs.microsoft.com/dotnet/csharp/programming-guide/classes-and-structs/partial-classes-and-methods) in the Annotation-based approach.
+
+# Object Types
+
+Given is the following entity that we want to extend with functionality.
+
+```csharp
+public class Book
+{
+    public int Id { get; set; }
+
+    public string Title { get; set; }
+
+    public int AuthorId { get; set; }
+}
+```
+
+## Adding fields
+
+We can easily add new fields to an existing type.
```

Comment:
```suggestion
We can easily add new fields to our existing `Booking` type.
```

Reply:
I was trying to have it flow from the top, but maybe this is overkill. Happy as you had it before too